utils.py

```diff
@@ -189,16 +189,63 @@ def validate_cookies_del(flows: list[dict]) -> None:
             )
 
 
-def flows_to_log(logger_fun: Callable, message: str, flow_dict: dict[str, list]):
-    """Log flows, maximun flows in a log is 200"""
-    length_msg = f"total_length: {len(flow_dict['flows'])}, "
+def _flows_to_log(
+    logger_fun: Callable,
+    flows: list,
+    dpid: Union[str, list],
+    counter: int,
+    flows_acc: dict,
+    maximun: int,
+    message: str,
+    total_length: str,
+) -> tuple[int, dict, str]:
+    start, end = 0, counter
+    while flows[start:end]:
+        counter -= len(flows[start:end])
+        flows_acc[str(dpid)] = flows[start: end]
```

Comment:
It's a bit hard to understand at first why it's accumulating by a potential str of list. I'd recommend you reviewing this and seeing if you can simplify or make it more readable.

Reply:
Method no longer necessary, removed.